src/plugins/liveobjects/liveobjects.ts

```
@@ -25,4 +36,142 @@
+    this._currentSyncCursor = undefined;
+    this._syncInProgress = false;
+  }
+
+  private _parseSyncChannelSerial(syncChannelSerial: string | null | undefined): {
+    syncId: string | undefined;
+    syncCursor: string | undefined;
+  } {
+    let match: RegExpMatchArray | null;
+    let syncId: string | undefined = undefined;
+    let syncCursor: string | undefined = undefined;
+    if (syncChannelSerial && (match = syncChannelSerial.match(/^([\w-]+):(.*)$/))) {
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)